shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/sql/supported/dml/select-sub-query.xml

```diff
@@ -18,7 +18,7 @@
 
 <sql-cases>
     <sql-case id="select_sub_query_with_project" value="SELECT order_id, (select 1) as num FROM t_order" />
-    <sql-case id="select_sub_query_with_table" value="SELECT t.* FROM (SELECT * FROM t_order WHERE order_id IN (?, ?)) t" />
+    <sql-case id="select_sub_query_with_table" value="SELECT a.* FROM (SELECT * FROM t_order WHERE order_id IN (?, ?)) a" />
```

Comment:
Is it necessary to use `a` instead of `t`?

Reply:
No, it isn't.